### DISCUSSION TITLE
Respond to pull request

### DISCUSSION THREAD

**Discussion by Chitusnair** [thawkes70/Tafe22S219#15] [Ideas]
Hi Tony,

I have reached up to section G - respond to pull request. If I raise a pull request can you please accept the request?

![image](https://user-images.githubusercontent.com/29492335/190425084-02b47f0a-281e-47cc-b45e-ce7e7d5dc202.png)

Thanks,

Chitra

**Comment by thawkes70** [comment]
Hi Chitra,

Sure can please go ahead.
